includes/gateway/class-omise-payment-installment.php

```diff
@@ -84,6 +85,26 @@ public function payment_fields()
 		);
 	}
 
+	/**
+	 * Get the total amount of an order
+	 */
+	public function getTotalAmount()
```

Comment:
Can we move this function under `Omise_Payment_Offsite` class to reuse if we needed for other payment methods in the future.

Reply:
We could but we don't enough information on this now. We are not sure if it will be applicable for offsite payments only. It might be needed for offline payment as well. In that case, `Omise_Payment` would be appropriate. Or we might never need it outside installment. So, I would suggest to do it when needed.